### PR TITLE
Add blogpost: The home offices of Sourcegraph

### PR DESCRIPTION
**Pull request by mrnugget**
This PR was caused by [this discussion in Slack](https://sourcegraph.slack.com/archives/C02FSM7DW/p1584979447231800).

So, if you want to participate and show off your home office setup:

1. Take a photo for your home office, resize to 1600px wide to reduce file-size.
1. Add your photo to https://console.cloud.google.com/storage/browser/sourcegraph-assets/about.sourcegraph.com/blog/?forceOnBucketsSortingFiltering=false&authuser=1&folder=true&organizationId=true&project=sourcegraph-dev
1. Edit the blog post markdown file in this PR and add your name, title and link to your photo.

Preview at https://deploy-preview-695--sourcegraph.netlify.com/blog/home-offices-of-sourcegraph

Deadline: **Friday, March 27th 2020, 18:00 CET / 10:00 PST**

### CODE REVIEW

**Comment by efritz** [comment]
This PR needs more JPEG.

File: blogposts/home-offices-of-sourcegraph.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+tags: [ blog, remote ]
+slug: home-offices-of-sourcegraph
+heroImage: /blog/home-offices-sourcegraph-thumbnail.jpg
+published: true
+---
+
+
+I'm always curious to see other people's desks (who isn't?) to get a glimpse of their setup, and after seeing the [home office desks of Basecamp](https://m.signalvnoise.com/remote-working-the-home-office-desks-of-basecamp/), I immediately wanted to have the same collection for Sourcegraph.
+
+Since we're an [all-remote company](https://about.sourcegraph.com/company/remote), we _only_ have home office setups and they range from "spare" to "elaborate", so all the more reason to reach out and start collecting photos.
```
Comment (christinaforney) [inline blogposts/home-offices-of-sourcegraph.md]:
```suggestion
As an [all-remote company](https://about.sourcegraph.com/company/remote), we _only_ have home office setups and they range from "sparse" to "elaborate", so all the more reason to create this collection of photos.
```

File: blogposts/home-offices-of-sourcegraph.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Here are all the photos that were contributed to the pull request behind this blog post, starting with mine:
```
Comment (christinaforney) [inline blogposts/home-offices-of-sourcegraph.md]: Is a blogpost the best format? Or do we want to create a page in our handbook, similar to the team page?

**Comment by slimsag** [comment]
Please do not commit binary files to this repository. It is trivial to create a bucket on Google Cloud and upload them and link them there.

**Comment by slimsag** [comment]
Such a bucket already exists, even: https://console.cloud.google.com/storage/browser/about.sourcegraph.com?forceOnBucketsSortingFiltering=false&authuser=1&folder=&organizationId=&project=sourcegraph-dev

**Comment by slimsag** [comment]
You might also consider running these images through mozjpeg (lossy) or jpegtran (lossless) to compress them.

**Comment by nicksnyder** [comment]
Might be nice to link to everyone's name to their entry on our team page in the handbook.

File: blogposts/home-offices-of-sourcegraph.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Ryan Blunden, Developer Advocate
+
+<!--I do a lot of screencasts and livestreams, hence the microphone, camera, and light. You might think the light is overkill, but it helps you look a lot more healthy and human when it's 4am Australian time.-->
+
+![Home office setup of Ryan Blunden, Developer Advocate](/blog/home-office-setups/ryan_blunden.jpg)
+
+## Christina Forney, Product Manager
+
+![Home office setup of Christina Forney, Product Manager](/blog/home-office-setups/christina_forney.jpg)
```
Comment (aileenrose) [inline blogposts/home-offices-of-sourcegraph.md]:
```suggestion
![Home office setup of Christina Forney, Product Manager](/blog/home-office-setups/christina_forney.jpg)

## Aileen Agricola, Senior Digital Marketing Manager

![Home office setup of Aileen Agricola, Senior Digital Marketing Manager](/blog/home-office-setups/aileen_agricola.jpg)
```

**Comment by mrnugget** [comment]
> Please do not commit binary files to this repository. It is trivial to create a bucket on Google Cloud and upload them and link them there.

I didn't know what the preferred way was but saw that other blog posts also have their images directly in this repository, so I thought I'd go with that.

From my personal experience I don't like binary files in repos either, but on the other hand I've seen blogposts work longer when the assets are right next to them and there's no danger of someone saying "what's that folder? looks like we don't need it, let's remove it" when going through cloud storage.

If we go with Google Cloud, though, I can move the assets to the same bucket that you used, though.

**Comment by nicksnyder** [comment]
@ryan-blunden Can you create some documentation on "How to create a blog post" that includes how images should be handled? If we don't want images checked in to this repo it would be nice to clean existing ones up so they don't set a bad example.

File: blogposts/home-offices-of-sourcegraph.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+![Home office setup of Stephen Gutekanst, Senior Software Engineer](https://storage.googleapis.com/sourcegraph-assets/about.sourcegraph.com/blog/home-office-setups/stephen_gutekanst.gif)
+
+## Farhan Attamimi, Software Engineer
+
+![Home office setup of Farhan Attamimi, Software Engineer](https://storage.cloud.google.com/sourcegraph-assets/about.sourcegraph.com/blog/home-office-setups/Farhan_Home_Office.JPG)
+
+## Rijnard van Tonder, Software Engineer
+
+![Home office setup of Rijnard van Tonder, Software Engineer](/blog/home-office-setups/rijnard_van_tonder.jpg)
```
Comment (dadlerj) [inline blogposts/home-offices-of-sourcegraph.md]:
```suggestion
![Home office setup of Rijnard van Tonder, Software Engineer](/blog/home-office-setups/rijnard_van_tonder.jpg)

## Dan Adler, VP Business

![Home office setup of Dan Adler, VP Business](https://storage.googleapis.com/sourcegraph-assets/about.sourcegraph.com/blog/home-office-setups/Dan_home_office.jpg)

```

**Pull request merged**